charts/redpanda/ci/96-audit-logging-values.yaml.tpl

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+enterprise:
+  license: "${REDPANDA_LICENSE}"
+
+auth:
+  sasl:
+    enabled: true
+    users:
+      - name: admin
+        password: change-me
+        mechanism: SCRAM-SHA-512
+
+# This should be changed to 23.3.0 or removed when we finally release this version for testing.
+image:
+  repository: docker.redpanda.com/redpandadata/redpanda-unstable
+  tag: v23.3.1-rc3
+
+auditLogging:
+  enabled: true
+  listeners: default
```

Comment:
Yes, but i rather test external as this one fails when we do not have a license in place.

Reply:
Why not both `default` and `internal`?

Reply:
you can only use one listener. I chose default since it fails consistently when license does not exist, which internal always worked.

Reply:
If your question is about testing the feature, we are only testing that we can functionally get things to work. Not to test the feature itself so one time showing things work is more than enough for us.

Reply:
> you can only use one listener. I chose default since it fails consistently when license does not exist, which internal always worked.

I didn't know that. Let's leave it.

> Not to test the feature itself so one time showing things work is more than enough for us.

Ok, but we don't have new test in https://github.com/redpanda-data/helm-charts/tree/main/charts/redpanda/templates/tests.

Reply:
Nothing much we can test, anytime something failed the chart failed to deploy. Any other test would be for functionality. If you want i can write a test to check that we do have the right topics, yeah i can do that.